### PR TITLE
BRS-1026: Fix QR code link to pass management, add ENV variable for l…

### PR DESCRIPTION
**Pull request by wilwong89**
…ink to pass management

### Jira Ticket:

BRS-1026

### Jira Ticket URL:

https://bcparksdigital.atlassian.net/browse/BRS-1026

### Description:

Adjusted the generated QR code for the admin link. Will add a new ENV variable to AWS parameter store. The new ENV variable will be `process.env.PASS_MANAGEMENT_ROUTE`

It is for the url the admin frontend. Example: `https://reserve-admin.bcparks.ca/dayuse/${PASS_MANAGEMENT_ROUTE}?=<some params>` 

Where `PASS_MANAGEMENT_ROUTE` should be added to the parameter store as `pass-management`. This new env variable will affect the admin frontend link generated QR codes.

### CODE REVIEW

**Comment by wilwong89** [comment]
Need to add new ENV variable to parameter store.

**Comment by wilwong89** [comment]
FYI: The new ENV variable will be `process.env.PASS_MANAGEMENT_ROUTE`

It is for the url the admin frontend. Example: `https://reserve-admin.bcparks.ca/dayuse/${PASS_MANAGEMENT_ROUTE}?=<some params>` 

Where `PASS_MANAGEMENT_ROUTE` should be added to the parameter store as `pass-management`. This new env variable will affect the admin frontend link generated QR codes.

**Review by cameronpettit — COMMENTED** [review]
If I'm not mistaken, if you're creating a new `env` variable to handle this, you'll need to instruct Terraform to grab it from the Parameter Store and inject it into the Lambda where it is used.  Check out how `params.tf` and `pass.tf` handle `envs`, and you can test with a sandbox deployment. 

If the parameter doesn't already exist in the target Parameter Store, Terragrunt will bork when this is deployed, so make sure dev, test, & prod have the correct params available before merging.

**Comment by wilwong89** [comment]
Updated `params.tf` and `pass.tf` to be able to pull ENV variables from AWS param store.

File: __tests__/writePass.test.js [hunk 1]
```diff
@@ -270,6 +270,8 @@ describe('Pass Successes', () => {
     const writePassHandler = require('../lambda/writePass/index');
     process.env.QR_CODE_ENABLED = "true";
     process.env.ADMIN_FRONTEND = "http://localhost:4300";
+    process.env.PASS_MANAGEMENT_ROUTE="/pass-management";
```
Comment (marklise) [inline __tests__/writePass.test.js:273]: Lint: Spaces around `=`

**Review by danieltruong — APPROVED** [review]

File: __tests__/writePass.test.js (hunk 1 above)
Comment (wilwong89) [reply __tests__/writePass.test.js:273]: Spaces added. I need to turn on my linter again. Sometimes it changes the entire file and I forgot to re-enable it.

**Pull request merged**